policy/policy-team-intro.md

```diff
@@ -0,0 +1,21 @@
+# CNCF Security SIG Policy Team Introduction
+
+## Background
+
+When SAFE WG was originally proposed, developers from Kubernetes Policy WG actively participated the discussions and then proposed CNCF level Policy WG was [merged](../policy-wg-merging.md) with SAFE WG to form the current CNCF Security SIG. Policy is also mentioned as part of the SIG [mission](../README.md).
+
+## Policy Team
+
+The Policy Team in CNCF Security SIG consists of the same set of developers that forms the Kubernetes Policy WG. The only difference is the topic coverage (whether it is at Kubernetes level or CNCF level).
```

Comment:
How do people know who is on the policy team?   How does someone join this team?  (Maybe something simple like people can self-identify by annotating their participation in ../README.md member list after they've participated in meetings and/or async participation via github)

Reply:
done in the updated patch